src/box/iproto.cc

```diff
@@ -130,6 +141,11 @@ struct iproto_thread {
 	struct cmsg_hop push_route[2];
 	struct cmsg_hop *dml_route[IPROTO_TYPE_STAT_MAX];
 	struct cmsg_hop connect_route[2];
+	/*
+	 * cmsg to notify tx thread about freeing of connection
+	 */
+	struct cmsg checking_alive_connections_msg;
+	int checking_alive_connections_status = CHECKING_NOT_START;
```

Comment:
It's better to not initialize field in struct definition - it's valid in c++11, but it's not used in our code in other places.

Reply:
Fix